lib/helpers.js

```diff
@@ -10,7 +10,18 @@ export const {
     CONTEXT_URL: CREDENTIALS_CONTEXT_V2_URL
   }
 } = credentialsContextV2;
+// Z and T must be uppercase
+// xml schema date time RegExp
+// @see https://www.w3.org/TR/xmlschema11-2/#dateTime
+export const dateRegex = new RegExp(
+  '-?([1-9][0-9]{3,}|0[0-9]{3})' +
+  '-(0[1-9]|1[0-2])' +
+  '-(0[1-9]|[12][0-9]|3[01])' +
+  'T(([01][0-9]|2[0-3]):[0-5][0-9]:[0-5][0-9](\.[0-9]+)?|(24:00:00(\.0+)?))' +
+  '(Z|(\\+|-)((0[0-9]|1[0-3]):[0-5][0-9]|14:00))?');
```

Comment:
add tests for offsets

Reply:
addressed here: https://github.com/digitalbazaar/vc/pull/158/commits/fb9791b94f6a04dde1da7adf622ddde09f28e2cd